packages/common/src/transforms/setMarks.ts

```diff
@@ -13,18 +13,13 @@ export const setMarks = (
 ) => {
   if (!editor.selection) return;
 
-  const clears: string[] = castArray(clear);
-  clears.forEach((item) => {
-    removeMark(editor, { key: item });
-  });
-
-  Object.keys(marks).forEach((key) => {
-    const markIsActive = isMarkActive(editor, key);
-
-    if (markIsActive) {
```

Comment:
Is removing this condition ok?

Reply:
I think it should be since, `unsetNodes` is basically a `setNode` that nulls the property.
On the other side of the if `deletet array['someNotExistingProp']` also returns with true.
So I don't think there is any issue with it.